src/manimShell.ts

```diff
@@ -17,6 +17,11 @@ const ANSI_CONTROL_SEQUENCE_REGEX = /(?:\x1B[@-Z\\-_]|[\x80-\x9A\x9C-\x9F]|(?:\x
  */
 const IPYTHON_CELL_START_REGEX = /^\s*In \[\d+\]:/gm;
 
+/**
+ * Regular expression to match IPython multiline input "...:"
+ */
+const IPYTHON_MULTILINE_START_REGEX = /^\s*\.{3}:\s+$/m;
+
```

Comment:
Could you add a few more words here explaining the context of why this is added? E.g. summarize the issue you had in a few sentences, explain what multiline input in IPython means and that this detection here is rather a hack as no easy solution was otherwise found. Just as a note for our future selves.